### ISSUE TITLE
Decode map chunk data

### ISSUE THREAD

**Issue by huin** [huin/chunkymonkey#6]
This isn't important for the core server, but potentially useful for anyone using the chunkymonkey/proto package for a client.

**Comment by jnwhiteh** [comment]
Could you provide any other information about what you want to happen here. It may be something isolated enough and of a size that I can take care of it.

**Comment by huin** [comment]
Yes, this is tiny and straightforward. It's probably only of use to code
that uses chunkymonkey/proto for client connections. The packet reader for
receiving map chunks doesn't extract the arrays.

**Comment by huin** [comment]
Point of note is that I've not done this yet as it's not been a priority. It's more for completeness of the chunkymonkey/proto package as both a server and client capable library. Strictly speaking, both readMapChunk and WriteMapChunk should be able to send subsets of a chunk, as described in the protocol doc - although this might needlessly complicate things, server-wise.
